src/main/java/com/moviesuggestion/model/Movie.java

```diff
@@ -0,0 +1,40 @@
+package com.moviesuggestion.model;
+
+public class Movie {
+    Long id;
```

Comment:
Make these fields private. It is one of the Java OOP: https://www.geeksforgeeks.org/encapsulation-in-java/